src/ui-lib/sass/06-utilities/_text.scss

```diff
@@ -1,3 +1,9 @@
 .grav-u-text-centered {
   text-align: center;
 }
+
+.grav-u-max-word-count-m {
+  max-width: gravy-breakpoint(regular);
```

Comment:
I think this ought to be in `em` rather than `px`, otherwise it defies the purpose of this piece of work.
Aside from this, I wouldn't even use the breakpoint as this can change in the future, and I'd rather introduce a new variable.

Reply:
I did use a new variable and then changed it. It was 40em before. I'll revert it to that.